crates/sui-framework/packages/deepbook/tests/clob_tests.move

```diff
@@ -766,6 +768,114 @@ module deepbook::clob_test {
         end(test)
     }
 
+    fun test_clean_up_expired_orders_(test: Scenario): TransactionEffects {
+        let (alice, bob) = people();
+
```

Comment:
Can we also have a scenario, or modify one of the scenarios where `alice` or `bob` place a mixture of buy and sell orders that expire? I am assuming a MM might use this function to clean up their expired orders and that would be a more realistic test.

Reply:
Sure, I will adjust the test code.